articles/synapse-analytics/sql-data-warehouse/maintenance-scheduling.md

```diff
@@ -21,12 +21,13 @@ For example, you can schedule a primary window of Saturday 22:00 to Sunday 01:00
 
 All newly created data warehouse instances will have a system-defined maintenance schedule applied during deployment. The schedule can be edited as soon as deployment is complete.
 
-Although a maintenance window can be between three and eight hours this does not mean the data warehouse will be offline for the duration. Maintenance can occur at any time within that window and you should expect a single disconnect during that period lasting ~5 -6 mins as the service deploys new code to your data warehouse. DW400c and lower may experience multiple brief losses in connectivity at various times during the maintenance window. When maintenance starts, all active sessions will be canceled, and non-committed transactions will be rolled back. To minimize instance downtime, make sure that your data warehouse has no long-running transactions before your chosen maintenance period.
+The duration of a maintenance window can range from three (3) to eight (8) hours, with a minimum requirement of three (3) hours. However, it is important to keep in mind that maintenance tasks typically take less than 30 minutes to complete, but it may take longer in some cases. For example, if there are active transactions when the maintenance starts, they will be aborted and rolled back, which may cause delays in coming back online. To avoid this scenario, we recommend that you ensure that no long-running transactions are active during the start of your maintenance window.
```

Comment:
This is still a bit confusing to the customer-- let's differentiate between the maintenance window options and the "maintenance window" as an event.  Perhaps something like this:
When selecting a maintenance window, you must choose a start time from the available options, as well as a maximum duration. The "maximum duration of a maintenance window" determines the time range within which maintenance will execute and can range from three (3) to eight (8) hours, with a minimum requirement of three (3) hours. During this window, your data warehouse will go offline for several minutes while your dedicated pool is moved to upgraded capacity through an operation similar to pause/resume.

Reply:
When choosing a maintenance window, you need to select a start time and set a maximum duration. The "maximum duration of a maintenance window" determines the timeframe in which maintenance tasks will be carried out. This timeframe can be between three (3) and eight (8) hours, with a minimum requirement of three (3) hours. During this period, your data warehouse will be temporarily offline as your dedicated pool is moved to upgraded capacity using a process similar to pause/resume. Under typical conditions, this operation will complete in well under 30 minutes, but it's important to note that in some cases, it can take longer. For instance, if there are active transactions when the maintenance begins, they will be canceled and rolled back, potentially causing delays in restoring the online service. To prevent this situation, we recommend ensuring that there are no long-running transactions active at the start of your maintenance window.